Makefile

```diff
@@ -48,11 +49,14 @@ ALL_ARCH = amd64 arm arm64 ppc64le s390x
 
 QEMUVERSION=v2.9.1-1
 
+BUSTED_ARGS=-v --pattern=_test
+
 IMGNAME = nginx-ingress-controller
 IMAGE = $(REGISTRY)/$(IMGNAME)
 MULTI_ARCH_IMG = $(IMAGE)-$(ARCH)
 
-BASEIMAGE=sagan/gladly-nginx:0.32
+# Set default base image dynamically for each arch
+BASEIMAGE=sagan/gladly-nginx:0.46
```

Comment:
this might be entirely wrong.

do we need to just use the upstream BASEIMAGE? -> https://github.com/kubernetes/ingress-nginx/blob/master/Makefile#L56